### ISSUE TITLE
[Bug Report] dev sleep command do not update on chain timestamp

### ISSUE THREAD

**Issue by jolestar** [starcoinorg/starcoin#3329]
# Bug Report

**Starcoin version:**

**Current behavior:**
```
chain info
dev sleep -t 10000
chain info
```

The on-chain timestamp does not change, the sleep command should trigger a new block.

**Expected behavior:**

the sleep command updates the on-chain timestamp.

**Steps to reproduce:**

**Related code:**

rpc/server/src/module/debug_rpc.rs

```
    fn sleep(&self, time: u64) -> Result<()> {
        if !self.config.net().is_test() && !self.config.net().is_dev() {
            return Err(jsonrpc_core::Error::invalid_request());
        }
        self.config.net().time_service().sleep(time);
        self.bus
            .broadcast(GenerateBlockEvent::new(true))
            .map_err(|_e| jsonrpc_core::Error::internal_error())?;
        Ok(())
    }
```

**Other information:**

**Comment by YusongWang** [comment]
This should close

**Comment by jolestar** [comment]
resolved by #3477
